### PR TITLE
Update chrome.zh-cn.md

### PR DESCRIPTION
**Pull request by cardamon-dk**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.

### Description

   Update chrome.zh-cn.md
### Motivation and Context

    Add some chineses translate
### Types of changes

- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [x] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [ ] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: website_and_docs/content/documentation/webdriver/browsers/chrome.zh-cn.md [hunk 1]
```diff
@@ -100,19 +97,19 @@ Add a browser location to options:
...
 {{< /tabpane >}}
 
-### Add extensions
+### 添加拓展程序
 
-The `extensions` parameter accepts crx files
+`extensions` 参数接受crx文件
 
-Add an extension to options:
+添加一个拓展程序到选项中:
```
Comment (alaahong) [inline website_and_docs/content/documentation/webdriver/browsers/chrome.zh-cn.md]: Actully, it called "扩展程序" as common transalation..
https://support.google.com/chrome_webstore/answer/2664769?hl=zh-Hans

File: website_and_docs/content/documentation/webdriver/browsers/chrome.zh-cn.md [hunk 2]
```diff
@@ -100,19 +97,19 @@ Add a browser location to options:
...
 {{< alert-code />}}
 
-### Keeping browser open
+### 保持浏览器的打开状态
 
-Setting the `detach` parameter to true will keep the browser open after the driver process has been quit.
+将 `detach` 参数设置为true将在驱动过程结束后保持浏览器的打开状态.
 
-Ad a binary to options:
+添加一个布尔值到选项中:
```
Comment (alaahong) [inline website_and_docs/content/documentation/webdriver/browsers/chrome.zh-cn.md:112]: Where is the boolean?

**Review by alaahong — CHANGES_REQUESTED** [review]
Alomst looks good, just comment on two special places...

File: website_and_docs/content/documentation/webdriver/browsers/chrome.zh-cn.md (hunk 2 above)
Comment (cardamon-dk) [reply website_and_docs/content/documentation/webdriver/browsers/chrome.zh-cn.md:112]: Well..  i try to understand the original english text and the use case below:

English text:
 `Ad a binary to options:`

seems like the `detach` parameter accept `0 | 1` 

Use case:
`  let driver = await env
        .builder().setChromeOptions(options.detachDriver(true)).build();`

It seems like the `detach` parameter accept `true | false`. So here is the boolean.. 

Seems like the english text  doesn't match the use case very well
So i changed the `binary` to `boolean` to match the use case
Let me know if this isn't appropriate or if I'm misunderstand

File: website_and_docs/content/documentation/webdriver/browsers/chrome.zh-cn.md (hunk 1 above)
Comment (cardamon-dk) [reply website_and_docs/content/documentation/webdriver/browsers/chrome.zh-cn.md]: > Actully, it called "扩展程序" as common transalation.. https://support.google.com/chrome_webstore/answer/2664769?hl=zh-Hans

Ok, got it !

File: website_and_docs/content/documentation/webdriver/browsers/chrome.zh-cn.md (hunk 2 above)
Comment (alaahong) [reply website_and_docs/content/documentation/webdriver/browsers/chrome.zh-cn.md:112]: I see, seems you are right, this is confusing by original English wording. Looks good, thanks

**Review by alaahong — APPROVED** [review]
Many thanks, looks good~

**Comment by harsha509** [comment]
Thank you for the review @alaahong !

**Comment by harsha509** [comment]
Congratulations on your first contribution to Selenium-docs @dk3817521  :tada: 🎉 🎉 🎉

**Pull request merged**